index.html

```diff
@@ -63,6 +63,9 @@
         functions such as qualifying new hardware. It's an open-source project
         under the GPL and is used and developed by a number of organizations,
         including Google, IBM, Red Hat, and many others.</p>
+        <p style="color:red;">Autotest in being discontinued in favor of the next
+        generation testing framework,
+        <a href="http://avocado-framework.github.io/">Avocado Framework</a>.</p>
```

Comment:
I like the brevity of your sentence, but "discontinued" may be too strong (I would leave this up to @lmr to decide the wording here, as I guess he's the official maintainer of Autotest). Maybe something like "Please check Avocado, a next generation test automation framework being developed by several of the original Autotest team members."

With that said, I suggest a notice be added to multiple places:

 - http://autotest.github.io/
 - README.rst (autotest.git)
 - documentation/source/index.rst (autotest.git)